MANIFEST.in

```diff
@@ -2,6 +2,7 @@ include README.md LICENSE CONTRIBUTORS.txt
 include openquake/engine/openquake.cfg
 include openquake/server/local_settings.py.pam
 include openquake/server/local_settings.py.standalone
+include openquake/hazardlib/gsim/mgmpe/*.csv
 
```

Comment:
I wonder if we should change that to `recursive-include openquake/hazardlib/gsim/*.csv` or even to  `recursive-include openquake/hazardlib/*.csv` ...

Reply:
Both will include also stuff from the tests... we can use them but then we need an explicit exclude for tests stuff.

What would you like to do?

Reply:
No, the first one will not include .csv from the tests, so I vote for that.

Reply:
I see. Will do.